_themes/kr/theme.conf

```diff
@@ -2,7 +2,8 @@
 inherit = basic
 stylesheet = flasky.css
 pygments_style = flask_theme_support.FlaskyStyle
+sidebars =
```

Comment:
this was mostly the problem with the build they added new sidebars to the basic theme here: https://github.com/sphinx-doc/sphinx/commit/b78f3090a05f38fb3beb14f7e716f1b26a2ff253#diff-8157ebb8774f8bad9cb7f2c97b2b46ec
setting it to empty seems to fix it